src/lightning_app/cli/lightning_cli.py

```diff
@@ -206,15 +290,63 @@ def stop():
     pass
 
 
-@_main.group(hidden=True)
+@_main.group(name="delete")
 def delete():
-    """Delete an application."""
+    """Delete a Lightning managed resources."""
     pass
 
 
-@_main.group(name="list", hidden=True)
+@delete.command("cluster")
+@click.argument("cluster", type=str)
+@click.option(
+    "--force",
+    "force",
+    type=bool,
+    required=False,
+    default=False,
+    is_flag=True,
+    help="""Delete a cluster from Lightning AI. This does NOT delete any resources created by the cluster,
+            it just removes the entry from Lightning AI.
+
+            WARNING: You should NOT use this under normal circumstances.""",
+)
+@click.option(
+    "--wait",
+    "wait",
+    type=bool,
+    required=False,
+    default=False,
+    is_flag=True,
+    help="Enabling this flag makes the CLI wait until the cluster is deleted.",
+)
+def delete_cluster(cluster: str, force: bool = False, wait: bool = False):
+    """Delete a Lightning AI compute cluster and all associated cloud provider resources.
+
+    Deleting a run also deletes all Runs and Experiments that were started on the cluster.
+    Deletion permanently removes not only the record of all runs on a cluster, but all associated experiments,
+    artifacts, metrics, logs, etc.
+
+    WARNING: This process may take a few minutes to complete, but once started it CANNOT be rolled back.
```

Comment:
Please change to:

WARNING: This process may take a few minutes to complete, but once started it CANNOT be rolled back.
    Deletion permanently removes not only the BYOC cluster from being managed by Lightning AI, but tears down
    every BYOC resource Lightning AI managed (for that cluster id) in the host cloud.

Reply:
:+1: